### PR TITLE
fix(instrumentation-aws-sdk): temporarily disable failing sqs test 

### PR DESCRIPTION
**Pull request by pichlermarc**
## Which problem is this PR solving?

See #1477. This test constantly failing reduces the visibility of failing tests in other packages. Skipping this test for now until #1477 is resolved. I'll add a comment in #1477 to remove the `skip()`.

### CODE REVIEW

**Comment by pichlermarc** [comment]
Looks like actually all 3 tests in `custom service behavior` failing.

**Review by trentm — APPROVED** [review]

**Review by david-luna — APPROVED** [review]

**Comment by pichlermarc** [comment]
Hmm, looks like now the tests are failing with `Error: Cannot find module '@smithy/hash-stream-node'` - I wonder if this is as similar issue to the one we're seeing in #1828

**Comment by trentm** [comment]
> Hmm, looks like now the tests are failing with `Error: Cannot find module '@smithy/hash-stream-node'` - I wonder if this is a similar issue to the one we're seeing in #1828

Explained here: https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1828#issuecomment-1834276719
tl;dr: `@aws-sdk/client-s3@3.377.0` was just a bad release.  #1828 will resolve it, because I've reduced the aws-sdk/client-s3 versions tested in the .tav.yml file, and explicitly skipped that broken version.

**Comment by trentm** [comment]
https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1847 is an alternative change that reduces the number of tests skipped.
